src/controllers/dynakube/dtclient_reconciler.go

```diff
@@ -96,21 +88,36 @@ func (r *DynatraceClientReconciler) Reconcile(ctx context.Context, instance *dyn
 	}
 
 	valid := true
+	updateSecret := false
+
+	if r.ApiToken == "" {
+		updateCR = setCondition(&sts.Conditions, metav1.Condition{
+			Type:    apiTokenConf.Type,
+			Status:  metav1.ConditionFalse,
+			Reason:  dynatracev1beta1.ReasonTokenMissing,
+			Message: fmt.Sprintf("Token %s on secret %s missing", apiTokenConf.Key, secretKey),
+		}) || updateCR
+		valid = false
+	} else if r.PaasToken == "" {
+		// if paas token is missing api token is paas token
```

Comment:
should it really be stored twice then?
could we add a flag on the dynakube status to indicate that api/paas token are the same or would that just complicate tings?

Reply:
Guess it would not even work, as we need the paasToken field in our secret.

E.g. [env_vars.go](https://github.com/Dynatrace/dynatrace-operator/blob/3dd79f8fbfca65cbdf0414503fe7f03c72c40068/src/controllers/dynakube/oneagent/daemonset/env_vars.go#L43) - The installer env var is grabbed from the secret. FYI: This is only used if the oneagent is not downloaded from the tenant.

So there has to be a paasToken field in the secret.

Reply:
ok, shouldn't be done for this pr then